### PR TITLE
3.0 add code highligthing in generated api

### PR DESCRIPTION
**Pull request by antograssiot**
Nothing related to the code itself, just changes how code example are rendered. before:
capture1

to
capture2

### CODE REVIEW

File: src/Controller/Component/AuthComponent.php [hunk 1]
```diff
@@ -537,9 +537,14 @@ public function getAuthorize($alias)
...
      * You can use allow with either an array or a simple string.
      *
-     * `$this->Auth->allow('view');`
-     * `$this->Auth->allow(['edit', 'add']);`
-     * `$this->Auth->allow();` to allow all actions
+     * ```
+     * $this->Auth->allow('view');
+     * $this->Auth->allow(['edit', 'add']);
+     * ```
+     * or to allow all actions
```
Comment (ADmad) [inline src/Controller/Component/AuthComponent.php]: A new line above and below the code fence is not required? I don't remember myself.

Comment (antograssiot) [inline src/Controller/Component/AuthComponent.php]: @ADmad no I've checked it. The only thing that's required is a new line between both fence so we can't use \`\`\`some code\`\`\` on the same line

Comment (ADmad) [inline src/Controller/Component/AuthComponent.php]: Okay, cool.

**Comment by markstory** [comment]
Nice work!

**Pull request merged**